site/_docs/reference.md

```diff
@@ -2647,6 +2647,7 @@ BigQuery's type system uses confusingly different names for types and functions:
 | m | expr1 <=> expr2                                | Whether two values are equal, treating null values as the same, and it's similar to `IS NOT DISTINCT FROM`
 | b | ARRAY_CONCAT(array [, array ]*)                | Concatenates one or more arrays. If any input argument is `NULL` the function returns `NULL`
 | b | ARRAY_LENGTH(array)                            | Synonym for `CARDINALITY`
+| s | ARRAY_DISTINCT(array)                          | Returns unique elements of *array*. Keeps ordering of elements.
```

Comment:
Should ARRAY_DISTINCT between ARRAY_CONCAT and ARRAY_LENGTH?
> I like to maintain dictionary order.

Reply:
fixed @JiajunBernoulli